File: includes/game.py
class Game():

    def __init__(self, team2, scoreDiff, next = None):
        self._team2 = team2
        self._scoreDiff = scoreDiff
        self.next = next

    def getTeam2(self): return self._team2
    def getScoreDiff(self): return self._scoreDiff

File: cpp/src/game.cpp
#include "game.h"

Game::Game(string team2, int scoreDiff, Game *next)
{
    this->team2 = team2;
    this->scoreDiff = scoreDiff;
    this->next = next;
}

string Game::getTeam2Name(void) { return this->team2; }
int Game::getScoreDiff(void) { return this->scoreDiff; }